src/lightning_app/core/api.py

```diff
@@ -346,34 +315,28 @@ async def check_is_started(self, queue):
 def start_server(
     api_publish_state_queue,
     api_delta_queue,
-    commands_requests_queue,
-    commands_responses_queue,
-    commands_metadata_queue,
```

Comment:
Is this a breaking change ? means that you can run apps on the current cloud with this branch ?